Library/Formula/gcc.rb

```diff
@@ -61,8 +61,8 @@ def osmajor
 
   # The bottles are built on systems with the CLT installed, and do not work
   # out of the box on Xcode-only systems due to an incorrect sysroot.
-  def pour_bottle?
-    MacOS::CLT.installed?
+  pour_bottle? "The bottle needs the Xcode CLT to be installed." do
+    MacOS::CLT.installed? && false
```

Comment:
The `&& false` looks like it's a leftover from testing this and doesn't belong here?